Formula/luajit.rb

```diff
@@ -35,7 +39,12 @@ def install
     ENV.O2 # Respect the developer's choice.
 
     args = %W[PREFIX=#{prefix}]
-    args << "XCFLAGS=-DLUAJIT_ENABLE_LUA52COMPAT" if build.with? "52compat"
+
+    cflags = ""
+    cflags += "-DLUAJIT_ENABLE_LUA52COMPAT " if build.with? "52compat"
+    cflags += "-DLUAJIT_ENABLE_GC64" if build.with? "gc64"
+
+    args << "XCFLAGS=#{cflags}" if cflags != ""
```

Comment:
I'm not sure I love this syntax choice but I'll leave that down to ILZ. If we're going to use it though something like `unless cflags.zero?` should work & looks nicer.